FAQ-Master.md

```diff
@@ -201,14 +201,33 @@ courses of action. OpenC2 can be utilized to populate STIX COA
 SDOs for sharing automated courses of action for the purpose of
 responding to cyber incidents in cyber-relevant time.
 
-### How does OpenC2 relate to [MISP](https://www.misp-project.org/features.html)? 
+### How does OpenC2 relate to [PACE](https://opencybersecurityalliance.org/pace/)?
+
+Posture Attribute Collection and Evaluation (PACE) is an
+[Open Cybersecurity Alliance](https://opencybersecurityalliance.org/)
+effort to create a
+comprehensive automated strategy for understanding security posture.
+OpenC2 plays two roles in PACE.
+PACE uses OpenC2 for command & control to obtain security posture information,
+and for the command & control of the PACE system itself
+(e.g. request securit posture evaluation of a device).
+OpenC2 also plays a role in the actions that result from posture evaluation
+(e.g. add a firewall rule, sandbox a device, etc).
+
+### How does OpenC2 relate to [Security Content Automation Protocol (SCAP)](https://csrc.nist.gov/projects/security-content-automation-protocol)?
```

Comment:
As I understand it, PACE has taken up where SCAPv2 was abandoned. But since basic [SCAP](https://en.wikipedia.org/wiki/Security_Content_Automation_Protocol) still exists and is used operationally, this very concise "next generation of SCAP" answer seems less than illuminating for those not "in the know".

Reply:
removed scap answer and changed to "to be specified"